nautobot/utilities/tests/test_filters.py

```diff
@@ -991,6 +992,48 @@ def test_device_comments_multiple_value_charfield_iregex_negation(self):
         self.assertEqual(self.DeviceFilterSetWithComments(params, self.device_queryset).qs.count(), 0)
 
 
+class GetFiltersetTestValuesTest(FilterTestCases.BaseFilterTestCase):
+    """Tests for `BaseFilterTestCase.get_filterset_test_values()`."""
+
+    queryset = Site.objects.all()
+    exception_message = "Cannot find valid test data for Site field description"
+
+    @classmethod
+    def setUpTestData(cls):
+        statuses = Status.objects.get_for_model(Site)
+        cls.status_active = statuses.get(slug="active")
+
+        Site.objects.all().delete()
```

Comment:
Do we have any models in or planned for our factories that have a PROTECT foreign key to Site, which would make this problematic? I wonder if instead of deleting all pre-existing Sites, we could instead use a filtered Site queryset for the below tests that only contains the explicitly tested Site(s)?

Reply:
We should. There are a lot of protected relationships to Site. Thanks for the heads up, I'll change this.